ops/readme.md

```diff
@@ -0,0 +1,20 @@
+# Operations
+This document describes how we operate the sites which are running Coronastatus.
+
+## Ownership
+
+| hostname            | country         | domain owner                           | server owner           |
+|---------------------|-----------------|----------------------------------------|------------------------|
+| coronastatus.nl     | The Netherlands | Adriaan Van Rossum / Simple Analytics  | Eirik Fosse / BustByte |
+| coronastatus.no     | Norway          | Eirik Fosse / BustByte                 | Eirik Fosse / BustByte |
+| coronastatus.es     | Spain           | Adriaan Van Rossum / Simple Analytics  | Eirik Fosse / BustByte |
+| coronastatus.it     | Italy           | Adriaan Van Rossum / Simple Analaytics | Eirik Fosse / BustByte |
+| coronastatus.mx     | Mexico          |                                        | Eirik Fosse / BustByte |
+| coronastatus.fr     | France          | Adriaan Van Rossum / Simple Analytics  | Eirik Fosse / BustByte |
+| coronastatus.us     | USA             |                                        | Eirik Fosse / BustByte |
+| coronastatus.co     | Colombia        |                                        | Eirik Fosse / BustByte |
+| coronastatus.com.br | Brazil          |                                        | Eirik Fosse / BustByte |
+| coronastatusau.org  | Australia       | Mauricio Wolff                         | Eirik Fosse / BustByte |
+| coronastatus.ng     | Nigeria         |                                        | Eirik Fosse / BustByte |
+| corona-status.in    | India           | Vandan Panwala                         | Eirik Fosse / BustByte |
+|                     |                 |                                        |                        |
```

Comment:
`| coronastatus.sk | Slovakia | Stano Bocinec / juniq.net | Stano Bocinec / juniq.net |`

Also thinking if it's not better to use our github aliases rather than names, to provide clear info :-) 
`| coronastatus.sk | Slovakia | @sbocinec  | @sbocinec  |`

(don't need my personal domain here though)